modules/common/util/lru_cache.h

```diff
@@ -169,12 +173,29 @@ class LRUCache {
     Init();
   }
 
+  bool Remove(const K& key) {
+    if (!Contains(key)) {
+      return false;
+    }
+    auto* node = &map_[key];
+    Detach(node);
+    return true;
+  }
+
+  bool ChangeCapacity(const size_t capacity) {
+    if (size() > capacity) {
+      return false;
+    }
+    capacity_ = capacity;
+    return true;
+  }
+
  private:
   static constexpr size_t kDefaultCapacity = 10;
 
   const size_t capacity_;
   size_t size_;
-  std::unordered_map<K, Node<K, V>> map_;
+  std::map<K, Node<K, V>> map_;
```

Comment:
For what considerations should we change `std::unordered_map` to `std::map` ?

Reply:
If use unordered_map, the key which isn't built-in type must provide a hash function to make the unique key, it is not convenient for us.